awx/main/models/events.py

```diff
@@ -456,38 +409,6 @@ def get_absolute_url(self, request=None):
     def __str__(self):
         return u'%s @ %s' % (self.get_event_display2(), self.created.isoformat())
 
-    def _update_from_event_data(self):
-        # Update job event hostname
```

Comment:
We no longer need any of this code, because is already in the event that goes across the message bus.